README.md

```diff
@@ -1,3 +1,20 @@
 # eds4jinja2
-Dynamic Data Context (DDC) with Embeded Datasource Specification (EDS) for Jinja2 templates
+Embedded Datasource Specification in Jinja2 templates
 
+Specify the data sources in your JINJA templates directly.
+
+
+```jinja2
+{% set content, error = from_file(path).fetch_tree() %} \n
+content:  {{ content }}\n
+error: {{ error }}\n
+```
+
+```jinja2
+{% set content, error =
+    from_endpoint(endpoint).with_query(query_string).fetch_tabular() %}
+content:  {{ content }} \n
+error: {{ error }} \n
+```
+
+[Read the docs here](https://eds4jinja2.readthedocs.io/en/feature-mdr-111/)  
```

Comment:
why the `/feature-mdr-111`?

Reply:
this is a test that it works. 
Will change now to the latest now